test.js

```diff
@@ -223,6 +223,17 @@ test('should ignore digits', (t) => {
     }, t.ifErr)
 })
 
+test('should ignore times', (t) => {
+  t.plan(1)
```

Comment:
```suggestion
  t.plan(2)
```

Let’s add another test.